user-storage-jpa/src/main/java/org/keycloak/quickstart/storage/user/EjbExampleUserStorageProviderFactory.java

```diff
@@ -46,7 +47,7 @@ public EjbExampleUserStorageProvider create(KeycloakSession session, ComponentMo
 
     @Override
     public String getId() {
-        return "example-user-storage-jpa";
+        return PROVIDER_ID;
```

Comment:
@vmuzikar if we are not referencing PROVIDER_ID anywhere inside the code, I believe that's fine to write:
```
public String getId() {
  return "example-user-storage-jpa";
}
```
But that's up to you and not a deal breaker for me.

Reply:
It is referenced in the test:
https://github.com/keycloak/keycloak-quickstarts/blob/1f74955b2d0b02a5dd711c3d6670f664fb04a061/user-storage-jpa/src/test/java/org/keycloak/quickstart/ArquillianJpaStorageTest.java#L147-L149
I know code changes should not be made just for testing purposes, but I felt that it's a best practise to have the provider ID as a public constant anyway, so I made this chage. But let me know if you want me to move this constant to the test class.